code/game/objects/structures/crates_lockers/closets/onestar.dm

```diff
@@ -0,0 +1,241 @@
+//Closets full of loot, they should be placed in derelicts
+
+//// Loot table on all tiers
+// Tier 1
+/obj/structure/closet/onestar/tier1
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, it may contain junk loot."
```

Comment:
desc = "Old OneStar closet, doesn't look like it contains anything worthwhile." or something along those lines. 
Sounds better, more IC-ly. 
Same for the closet descriptions below.